### PR TITLE
Update for several commands in BelayBox cheat sheet.

### PR DESCRIPTION
**Pull request by krealyt**
Did some updates in the BelayBox Cheat Sheet.

Signed-off-by: Krealyt <manuel.ziegler@pionix.de>

### CODE REVIEW

**Review by pionix-juraed — COMMENTED** [review]
Line 77: I would delete the sentence: "Format this partition to reset [...]" It is not really recommended to format this partition.

**Review by pionix-juraed — COMMENTED** [review]
Line 95: Delete this part as we do not plan to have partial updates. " ...partial updates
are not implemented yet."

File: docs/misc/pionix_belay_box.rst [hunk 1]
```diff
@@ -212,27 +212,15 @@ Reference Cheat sheet
...
-    config containing only the diffs to the default config.
-* to stop automatic updates: rw; sudo systemctl disable ota-update.timer
-* /mnt/user_data/etc/mosquitto/conf.d: 
-    here you can add additional config files
-    for the mqtt broker. For example a “public_mqtt.conf” file with the 
-    following contents:
+* /mnt/user_data/etc/wpa_supplicant/wpa_supplicant.conf: file containing wifi settings
+* /mnt/user_data/opt/everest/<crosscompiled everest binaries> force the use of custom everest build or config by automated start of ``everest-dev.service`` instead of ``everest.service``
+* /mnt/user_data/etc/update_channel contains either stable or unstable to define release channels
+* /mnt/user_data/etc/wireguard/<wireguard interface name>.conf for a wireguard VPN configuration
```
Comment (pionix-juraed) [inline docs/misc/pionix_belay_box.rst]: replace "wireguard interface name"  with "wg0.conf" as it is not freely selectable.

**Review by pionix-juraed — CHANGES_REQUESTED** [review]
See comments for the required changes.

**Comment by krealyt** [comment]
@pionix-juraed Changed your requests. Please re-review.

**Review by pionix-juraed — APPROVED** [review]
looks good!

**Pull request merged**